packages-exp/app-exp/src/public-types.ts

```diff
@@ -70,16 +70,48 @@ export interface FirebaseApp {
 /**
  * @public
  *
- * Firebase configuration object
+ * Firebase configuration object. Contains a set of parameters required by
+ * services in order to successfully communicate with Firebase server APIs
+ * and to associate client data with your Firebase project and
+ * Firebase application.
  */
 export interface FirebaseOptions {
+  /**
+   * An encrypted string used when calling certain APIs that don't need to
+   * access private user data
+   * (example value: `AIzaSyDOCAbC123dEf456GhI789jKl012-MnO`).
+   */
   apiKey?: string;
+  /**
+   * Auth domain for the project ID.
+   */
   authDomain?: string;
+  /**
+   * Default Realtime Database URL.
+   */
   databaseURL?: string;
+  /**
+   * The unique identifier for the project across all of Firebase and
+   * Google Cloud.
+   */
   projectId?: string;
+  /**
+   * The default Cloud Storage bucket name.
+   */
   storageBucket?: string;
+  /**
+   * Unique numerical value used to identify each sender that can send
+   * Firebse Cloud Messaging messages to client apps.
+   */
   messagingSenderId?: string;
+  /**
+   * Unique identifier for the app.
+   */
   appId?: string;
+  /**
+   * An ID automatically created when you enable Analytics in your
```

Comment:
Should we mention that measurementId is optional in v7.20.0 and up?

Reply:
Done.